tests/foreman/ui/test_errata.py

```diff
@@ -93,7 +92,7 @@ def _org():
         value='Yes',
         organization=org.id,
     ).create()
-    upload_manifest_locked(org.id)
+    module_target_sat.upload_manifest(org.id)
```

Comment:
`upload_manifest_locked` performs the manifest cloning functionality, so this will fail unless we either pass a manifest to `upload_manifest` or just use `module_entitlement_manifest_org` instead of `manifest_org`.

Reply:
I made the changes in upload_manifest method itself and it's working.

Reply:
Ok, so it's currently working with a cloned manifest. That should be fine for now, and I'll convert it to manifester when I get to this module in https://github.com/SatelliteQE/robottelo/pull/10394.